sdk/src/Services/EC2/Custom/Internal/AmazonEC2ResponseHandler.cs

```diff
@@ -219,39 +225,42 @@ private static void RestoreRequestIpPermissions(List<IpPermission> IpPermissions
         {
             foreach (var ipPermission in IpPermissions)
             {
-                if(ipPermission.RestoreOldIpV4Range)
+                if (ipPermission.RestoreOldIpV4Range)
```

Comment:
No null check here since `RestoreRequestIpPermissions` is only invoked when `IpPermissions` is not null.